src/Raven.Client/Documents/Operations/Replication/DetailedReplicationHubAccess.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace Raven.Client.Documents.Operations.Replication
+{
+    public class DetailedReplicationHubAccess
+    {
+        public string Name;
+        public string Thumbprint;
+        public DateTime NotBefore, NotAfter;
+        public string Subject;
+        public string Issuer;
+        
+        public string[] Incoming;
+        public string[] Outgoing;
```

Comment:
For consistency, is it possible to have here also `AllowedWritePaths `& `AllowedReadPaths `?